### PR TITLE
Turns Foxtrot's radio channel on by default

### PR DESCRIPTION
**Pull request by theselfish**
# About the pull request

image

Why is this off by default for *everyone*?

# Explain why it's good for the game

When you wake up Foxtrot, there shouldn't be a lack of communication.
Having to beg command to turn your channel on just slows everything down.

I left it off for keys that typically have squad channels off (CMP, CE, RO, CT, etc)

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
qol: Foxtrot's radio channel is now on if you have multi-squad in your headset.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -306,7 +306,7 @@
 /obj/item/device/encryptionkey/cmb
 	name = "\improper Colonial Marshal Bureau Radio Encryption Key"
 	icon_state = "cmb_key"
-	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE)
```
Comment (harryob) [inline code/game/objects/items/devices/radio/encryptionkey.dm:309]: why was this removed

Comment (theselfish) [reply code/game/objects/items/devices/radio/encryptionkey.dm:309]: Hu-

I'm guessing my branch was out of date. Give me a moment

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -306,7 +306,7 @@
...
+	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE)
```
Comment (theselfish) [inline code/game/objects/items/devices/radio/encryptionkey.dm:309]:
```suggestion
	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE))
```

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 3]
```diff
@@ -306,7 +306,7 @@
 /obj/item/device/encryptionkey/cmb
 	name = "\improper Colonial Marshal Bureau Radio Encryption Key"
 	icon_state = "cmb_key"
-	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE)
+	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE))
```
Comment (theselfish) [inline code/game/objects/items/devices/radio/encryptionkey.dm:309]:
```suggestion
	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE)
```

**Comment by theselfish** [comment]
@harryob Sorry about that. Oversight.

**Review by BeagleGaming1 — APPROVED** [review]
code seems good
